nginx.conf

```diff
@@ -20,20 +20,34 @@
 		ssl_trusted_certificate	/srv/nginx/ca-certs.pem;
 		ssl_dhparam		/srv/nginx/dhparam.pem;
 
-		# Authentification
-		set			$setup_type ovf-ami;
-		set			$realm off;
-
-		if (-f /srv/nginx/.htpasswd) {
-			set		$realm on;
-			set		$setup_type "${setup_type}-protected";
-		}
-		if ($setup_type = "ovf-ami") {
-			rewrite		^/$ $scheme://$http_host/password-page/ permanent;
-		}
-		if ($setup_type ~ "-protected") {
-			rewrite		^/password-page $scheme://$http_host/ permanent;
-		}
+        # Authentification
+        set $setup_type ovf;
+        set $realm off;
+
+        if (-f /.pmm-docker) {
+            set $setup_type docker;
+        }
+
+        if (-f /.pmm-ami) {
+            set $setup_type ami;
+        }
+
+        if (-f /srv/nginx/.htpasswd) {
+            set $realm on;
+            set $setup_type "${setup_type}-protected";
+        }
+        if ($setup_type = "docker") {
+            rewrite ^/$ $scheme://$http_host/password-page/docker/ permanent;
```

Comment:
We could get rid of this `if` by moving the redirect to line 28. The same applies to the other two `ifs`.